easybuild/easyconfigs/c/Cantera/Cantera-2.4.0-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'SCons'
+
+name = 'Cantera'
+version = '2.4.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/Cantera/cantera'
+description = """Chemical kinetics, thermodynamics, and transport tool suite"""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'strict': True}
+
+source_urls = ['https://github.com/Cantera/cantera/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['0dc771693b657d8f4ba835dd229939e5b9cfd8348d2f5ba82775451a524365a5']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('Boost', '1.66.0'),
+    ('SUNDIALS', '2.7.0'),
+]
+builddependencies = [
+    ('SCons', '3.0.1', versionsuffix),
+    ('Eigen', '3.3.4', '', True),
+    ('fmt', '3.0.2'),
+    ('googletest', '1.8.0'),
+]
+
+common_opts = 'env_vars=all CC="$CC" CXX="$CXX" cc_flags="$CFLAGS" cxx_flags="$CXXFLAGS" '
+common_opts += 'blas_lapack_libs=mkl_rt blas_lapack_dir=$BLAS_LAPACK_LIB_DIR '
+common_opts += 'sundials_include=$EBROOTSUNDIALS/include sundials_libdir=$EBROOTSUNDIALS/lib'
+buildopts = 'build ' + common_opts
+# tests hang, so disable them for now
+# runtest = 'test ' + common_opts
+installopts = 'install ' + common_opts
+prefix_arg = 'prefix='
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
+
+sanity_check_paths = {
+    'files': ['bin/ck2cti'],
+    'dirs': ['include/cantera', 'lib/pkgconfig', 'lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = [('python', "-c 'import cantera'")]
```

Comment:
@boegel isn't this check run anyway, if the module name is set correctly?

Reply:
@verdurin That's only done automatically when the `PythonPackage` easyblock (or one based on top of that) is used, not when `SCons` is used...

Reply:
@boegel of course. Thanks for clarifying.